app/assets/javascripts/faq.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  var faqContainer = $('.faq-row');
```

Comment:
Any thoughts on using data-roles, or `js-` prefixes, to separate concerns for styling an interaction?